app/javascript/src/components/Dashboard/Notes/NotesMenuBar.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "@bigbinary/neeto-icons";
+import { Typography } from "@bigbinary/neetoui";
+import { MenuBar } from "@bigbinary/neetoui/layouts";
+
+export default function NotesMenuBar({ showMenu }) {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <div>
+      <MenuBar showMenu={showMenu} title="Notes">
+        <MenuBar.Block label="All" count={200} active />
+        <MenuBar.Block label="Users" count={80} />
+        <MenuBar.Block label="Leads" count={60} />
+        <MenuBar.Block label="Visitors" count={60} />
+
```

Comment:
Please make sure to carefully read the `React Best Practices` section from the `Learn React` course in BBA.

https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Don%E2%80%99t%20add%20unnecessary%20blank%20lines

Reply:
Done.Thanks